d10-queries.sql

```diff
@@ -3,8 +3,8 @@
 UPDATE `taxonomy_term_field_data` SET `description__format`='full_html' WHERE vid = 'blog_type' and langcode = 'fr'
```

Comment:
Not all changes from e2214bb5 were copied over.

Missing semicolon on line 3. Comments are still PHP-style, not SQL-style.

Reply:
These changes are important, because I run these queries through Drush

    drush sql:query --database=default --file=/var/www/yukon-ca-d10/d10-queries.sql